docs/releases/slashing-release/interfaces/core/slashing-parameter-configuration.md

```diff
@@ -0,0 +1,222 @@
+# Slashing Parameter Configuration
+
+Allocation functionality and magnitudes are explained [here](https://www.notion.so/eigen-labs/Allocator-Functionality-282a008ab7a14c79a25ec2954f8f5912).
+
+```solidity
+interface IAVSDirectory {
+    /// STRUCTS
+
+    /**
+     * @notice Struct representing the registration status of an operator with an operator set.
+     * Keeps track of last deregistered timestamp for slashability concerns.
+     * @param registered whether the operator is registered with the operator set
+     * @param lastDeregisteredTimestamp the timestamp at which the operator was last deregistered
+     */
+    struct OperatorSetRegistrationStatus {
```

Comment:
move this to operator-sets.md imo

Reply:
https://github.com/Layr-Labs/eigenlayer-contracts/pull/709/commits/df5dd0a437851e2dcd792e17b6c5146bee48fec4